project/AppDependencies.scala

```diff
@@ -6,11 +6,11 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "3.4.0"
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "5.3.0"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "3.4.0"                % Test,
```

Comment:
The catalogue currently doesn't call out test dependancies that are behind but some of the versions here could be updated. Please see https://catalogue.tax.service.gov.uk/dependencyexplorer

Reply:
I've updated the bootstrap. I've also checked the other test dependencies using the link you provided and all seem to be up-to-date

Reply:
I've updated the bootstrap. I've also checked the other test dependencies using the link you provided and all seem to be up-to-date

Reply:
Please see the below. Mike and I walked through them yesterday.

https://github.com/hmrc/income-tax-interest/commit/bdd3862c1a6c1accf848a739c10a4fb421111a99#diff-34ef524c498e830d38453638b16bce7018ebcdb7e360316a2b7f437b49d9dee1

Reply:
Thanks, done